docs/docs/b2c/azure_setup.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: Azure configuration
+sidebar_position: 1
+---
+
+We'll need to create two application registrations for Azure AD B2C authentication to cover both direct API
+use and usage from the OpenAPI (swagger) documentation.
+
+:::info
+This guide assumes that an Azure B2C tenant was already created and linked to an Azure subscription.
+:::
+
+
+## Backend API
+
+### Step 1 - Create app registration
+Head over to
+[Azure -> Azure AD B2C -> App registrations](https://portal.azure.com/#view/Microsoft_AAD_B2CAdmin/TenantManagementMenuBlade/~/registeredApps),
+and create a new registration.
+
+Select a fitting name for your project; Azure will present the name to the user during consent.
+
+* `Supported account types`: `Accounts in any identity provider or organizational directory (for authenticating users with user flows)`
+* `Redirect URI`: Choose `Web` and `http://localhost:8000/signin-oidc` as a value
+* `Grant admin consent to openid and offline_access permissions`: `Yes`
+
+Press **Register**
+
+![1_application_registration](../../static/img/b2c/1_application_registration.png)
+
+### Step 2 - Verify token version is `v2`
+
+First we'll verify that the token version is version 2. In the left menu bar, click `Manifest` and find the line
+that says `accessTokenAcceptedVersion`. Verify that the value is `2`.
+
+Press **Save**
+
+(A change can take some time to happen, which is why we do this first.)
+
+![2_manifest](../../static/img/b2c/2_manifest.png)
+
+
+### Step 3 - Note down your tenant name and application ID
+
+Go back to the App Registration `Overview`, found in the left menu.
+
+![3_overview](../../static/img/b2c/3_overview.png)
+
+Copy the `Application (Client) ID`, we'll need it for later. I like to use `.env` files to
+store variables like these:
+
+```bash title=".env" {2}
+TENANT_NAME=
```

Comment:
I wish this was `TENANT_ID` instead of `TENANT_NAME` everywhere, to match the rest of the documentation and the demo_project shipped with this app. Not a big deal, though.

Reply:
@JonasKs I can change it, but I would understand `TENANT_ID` to be the GUID version of the tenant, not the "name" as used e.g. in scopes or other URLs, example:

```python
url = f'https://{settings.TENANT_NAME}.onmicrosoft.com/{settings.APP_CLIENT_ID}/user_impersonation'
```

I could use another specifier. I was using name, the UI uses "Initial domain name" https://learn.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-tenant#create-an-azure-ad-b2c-tenant

Reply:
Aha, I see. 

Your call. I'll merge when you're happy 😊